hypervisor/virt/ahv/ahvapi.py

```diff
@@ -351,14 +361,14 @@ def get_vm_host_uuid_from_vm(self, vm_entity):
 
     def get_vm(self, uuid):
```

Comment:
It seems this function is useless... already have the ```get_vm_by_name```

Reply:
yep, we don't need it right now, but what if we have the vms have the same name? I believe we will need it in the future for the multiple vms situation